ui/src/app/common/services/agents/agent.policies.service.ts

```diff
@@ -167,4 +169,233 @@ export class AgentPoliciesService {
         },
       );
   }
+
```

Comment:
@dscabral these are the mocked data I'm using for the agent policies service here in FE before we integrate with BE